src/mpid/ch4/netmod/ofi/errnames.txt

```diff
@@ -107,3 +107,5 @@
 **ofi_max_conn %d:OFI dynamic process reaches maximum connection (%d)
 **ofi_num_nics:OFI has a different number of nics available on different notes
 **ofi_num_nics %d %d:OFI has a different number of nics available on different notes (%d on this node and %d elsewhere)
+**ofid_mr_key:OFI MR key invalid
+**ofid_mr_key %s %d %s %s:OFI MR key invalid (%s:%d:%s:%s)
```

Comment:
One thing I didn't entirely understand is how're these format specifiers decided upon? @hzhou

Reply:
Think about how would you like (as the user) to read the error message. The error message don't have to include specific data if the message itself is clear enough, in which case you can omit the specific format (the one with formats) and just use e.g. `MPIR_ERR_CHKANDJUMP`.

Reply:
I see, okay. Thanks Hui.